CODE.md

```diff
@@ -0,0 +1,114 @@
+
+Here we will talk about [Codacy](https://app.codacy.com/project/gctools-outilsgc/concierge/dashboard) and how we will use it.
+
+Explain how it will run on each commit / pull request and must pass before we merge the pull request.
+Explain how we have our own code standards configured in Codacy based on the documentation below.
+
+We should talk about code test coverage that may also be tested by Codacy.
+
+I'd like to discuss how to use and install linters and configs for linters. Maybe I'll discuss that in each language section. :fire:
+
+## Languages :sparkles:
+Here are the code standards for different languages. We also have links to linters that are supported by Codacy. You can add these linters to your projects. :+1:
+
+### PHP :older_man:
+
+#### Code Style
+* [PEAR](https://pear.php.net/manual/en/standards.php)
+#### Linter
+* [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer)
+#### Test Coverage
+* [PHP Code Coverage for Codacy](https://github.com/codacy/python-codacy-coverage)
+
+### Javascript :godmode: :trophy:
+
+#### Code Style
+* [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)
+#### Linters / Config
+* [ESLint](https://eslint.org/)
+* [GCTools ESLint Config Package](https://github.com/gctools-outilsgc/gctools-components/tree/master/packages/eslint-config)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/codacy/node-codacy-coverage)
+
+### Python :snake:
+
+#### Code Style
+* [PEP 8](https://www.python.org/dev/peps/pep-0008/)
+#### Linter
+* [Pylint](https://www.pylint.org/)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/codacy/python-codacy-coverage)
+
+### CSS / Sass :art:
+
+#### Code Style
+* [Airbnb Style Guide](https://github.com/airbnb/css)
+#### Linters
+* [CSSLint](https://github.com/CSSLint/csslint)
+* [Stylelint](https://stylelint.io/)
+
+### Go :rat: (no gopher emoji :cry:)
+
+#### Code Style
+* [Effective Go](https://golang.org/doc/effective_go.html)
+* [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments)
+#### Linter
+* [Golint Repo](https://github.com/golang/lint)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/schrej/godacov)
+
+### Dockerfile :whale:
+
+#### Style
+* [Dockerfile Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
+#### Linter
+* [Hadolint](https://github.com/hadolint/hadolint)
+
+## Document Naming Convention :surfer:
+
+### Design Working Files (.ai .psd .pdf) / Images / Folders / Git Branches :page_facing_up:
+
+:clap: ALL LOWERCASE :clap:
+:clap: Underscores '_' instead of spaces or dashes :clap:
+
+#### Versioning :clipboard:
+Major versions 
+* v01
+* v02
+* v03
+* etc
+
+Minor changes to version
+* v01_1
+* v01_2
+* etc
+
+A working design document on its second major version with a minor change:
+blog_home_v02_1.ai

```

Comment:
How would this work with git? Would each version replace the previous one? You make a change to blog_home_v02.ai, and you'd have to rename it to blog_home_v02_1.ai, and push that (essentially delete blog_home_v02.ai and add blog_home_v02_1.ai)?

Reply:
@piet0024 is versioning a necessary evil for Design Docs when using git?

Reply:
The only issues with design docs and git is that there are non-plain text files involved and that prevents it from generating diffs for them, that doesn't prevent it from tracking different version like it does for other files though. I think the versions in the file name are more for those not used to git?

Reply:
This is a good point, I hadn't thought of github's ability to track the versions of documents. This style of versioning can be helpful for designers / individuals who are not familiar with git to keep track of the latest iterations on their design or if the designers are using local or shared drives. 

Are we still discussing how / if we will save design documents to github?

Reply:
Is there a very good reason why we shouldn't?